docs/install/installing-istio.md

```diff
@@ -0,0 +1,281 @@
+    --set sidecarInjectorWebhook.enabled=false \
+    --set global.proxy.autoInject=disabled \
+    --set global.omitSidecarInjectorConfigMap=true \
+    --set global.disablePolicyChecks=true \
+    --set prometheus.enabled=false \
+    `# Disable mixer prometheus adapter to remove istio default metrics.` \
+    --set mixer.adapters.prometheus.enabled=false \
+    `# Disable mixer policy check, since in our template we set no policy.` \
+    --set global.disablePolicyChecks=true \
+    `# Set gateway pods to 1 to sidestep eventual consistency / readiness problems.` \
+    --set gateways.istio-ingressgateway.autoscaleMin=1 \
+    --set gateways.istio-ingressgateway.autoscaleMax=1 \
+    `# Set pilot trace sampling to 100%` \
+    --set pilot.traceSampling=100 \
+    install/kubernetes/helm/istio \
+    `# Removing trailing whitespaces to make automation happy` \
+    | sed 's/[ \t]*$//' \
+    > ./istio-lean.yaml
+
+  kubectl apply -f istio-lean.yaml
+  ```
+
+### Installing Istio with Secret Discovery Service
+
+[Secret Discovery Service](https://istio.io/docs/tasks/traffic-management/secure-ingress/sds/)
+is needed if you want to dynamically update your Gateway  with multiple TLS
+certificates to terminate TLS connection. The below `helm` flag is needed in
+your `helm` command to enable `SDS`:
+
+```
+--set gateways.istio-ingressgateway.sds.enabled=true
+```
+
+For example, the `helm` command for installing Istio with Ingress `SDS` and 
+Istio sidecar injection is:
+
+```shell
+helm template --namespace=istio-system \
+  --set sidecarInjectorWebhook.enabled=true \
+  --set sidecarInjectorWebhook.enableNamespacesByDefault=true \
+  --set global.proxy.autoInject=disabled \
+  --set global.disablePolicyChecks=true \
+  --set prometheus.enabled=false \
+  `# Disable mixer prometheus adapter to remove istio default metrics.` \
+  --set mixer.adapters.prometheus.enabled=false \
+  `# Disable mixer policy check, since in our template we set no policy.` \
+  --set global.disablePolicyChecks=true \
+  `# Set gateway pods to 1 to sidestep eventual consistency / readiness problems.` \
+  --set gateways.istio-ingressgateway.autoscaleMin=1 \
+  --set gateways.istio-ingressgateway.autoscaleMax=1 \
+  --set gateways.istio-ingressgateway.resources.requests.cpu=500m \
+  --set gateways.istio-ingressgateway.resources.requests.memory=256Mi \
+  `# Enable SDS in the gateway to allow dynamically configuring TLS of gateway.` \
+  --set gateways.istio-ingressgateway.sds.enabled=true \
+  `# More pilot replicas for better scale` \
+  --set pilot.autoscaleMin=2 \
+  `# Set pilot trace sampling to 100%` \
+  --set pilot.traceSampling=100 \
+  install/kubernetes/helm/istio \
+  `# Removing trailing whitespaces to make automation happy` \
+  | sed 's/[ \t]*$//' \
+  > ./istio.yaml
+
+  kubectl apply -f istio.yaml
+
+```
+
+### Updating your install to use cluster local gateway
+
+If you want your Routes to be visible only inside the cluster, you may
```

Comment:
@ZhiminXiang 
Is this an "update"? IE, if you run this command when you already have Istio installed, will it update your install to add a new gateway?